.github/workflows/maven-test-compile.yml

```diff
@@ -0,0 +1,41 @@
+on:
```

Comment:
Can we use the same template as in other projects? Goal is actually to make provisioning of standard flows via code.

Reply:
The thing is, that this action is using phase `test-compile` (just to verify compilation of the code and tests) unlike [maven-test.yaml](https://github.com/wultra/wultra-infrastructure/blob/develop/.github/workflows/maven-test.yml) using phase `test`. I did not use `test` here, because sure-fire plugin is launching integration backend tests. These tests require several secrets. They should be reconfigured to integration tests and to be run by fail-safe plugin in later phase. This is a quick solution which can be improved in another PR.

Reply:
Ok, understood.